pkg/apis/feature/features.go

```diff
@@ -151,6 +156,19 @@ func (e Flags) OIDCDiscoveryBaseURL() string {
 	return string(discoveryUrl)
 }
 
+func (e Flags) RequestReplyDefaultTimeout() string {
+	if e == nil {
+		return string(DefaultRequestReplyTimeout)
+	}
+
+	timeout, ok := e[RequestReplyDefaultTimeout]
+	if !ok {
+		return string(DefaultRequestReplyTimeout)
+	}
+
+	return string(timeout) // TODO, check this, check this whole function really
```

Comment:
Instead of leaving a TODO here, can you track this in an issue?

Reply:
I forgot to remove this when I committed, sorry - I made a new commit to fix this